README.md

```diff
@@ -16,15 +16,27 @@ API resources. Some widely used annotations and/or CRDs _may_ be supported, as
 long as they can be translated to Gateway API directly. This project is not
 intended to copy annotations from Ingress to Gateway API.
 
-## Usage
+## Install
 
 This project reads Ingress resources from a Kubernetes cluster based on your
 current Kube Config. It will output YAML for equivalent Gateway API resources
 to stdout. Until this project is released, the best way to use this is to run
 the following within the repo:
 
+```shell
+curl -fsSL https://raw.githubusercontent.com/kubernetes-sigs/ingress2gateway/main/tools/hack/install.sh | bash 
```

Comment:
Hey @robscott, I think we should have both ways to install i2gw. This is related to Uer Experience, from my experience in istio and istioctl, we definitely need as diversity ways to install i2gw as possible. So three ways we have mentioned:
1. install i2gw from release page with a user friendly install script. Then user can install it as a binary command line tool.
2. install i2gw with go get and go install if users are familiar with go and have to local environment.
3. install i2gw as a krew addon for users who like using krew to manage kubcectl addons.

Reply:
I'm also very -1 on `curl | bash` installations. It's a _terrible_ habit for users to get into, since very few people actually spend the time to verify the signature of the downloaded scripts.

Okay, I'll grant it's better than `curl | sudo bash`, but not by much. I think that having:
- a binary on the releases page
- support for `go install`
- and a krew plugin

should be plenty.

Reply:
Thanks! I have removed the install script.